Source/TextToSpeechV1/Models/SynthesisVoice.swift

```diff
@@ -17,6 +17,7 @@
 import Foundation
 
 /** A voice to be used for synthesis. */
+// swiftlint:disable identifier_name
```

Comment:
This is an interesting case to consider with generation. We might have to add `swiftlint:disable` to all enums, since we won't be able to control the names well.

Reply:
I think we can control the enum names, as long as we map them to the appropriate values coming from the service.  But we'll definitely need to pay special attention to this.